website/docs/events/m1.md

```diff
@@ -40,7 +42,125 @@ Please consider topics such as:
+RD: This was our motivation.
+RD: We usualy receive feedback from students that we have outdated content, information that is repetitive or even mistakes. They do not have write access to OCW and they cannot be granted write access because OCW does not have a method for reviewing materials.
+RD: Usually content is private or, if it is publice, it's very difficult tu update.
+RD: It's not easy to create super high quality content, you need a lot of iterations.
+RD: We want to have as many actions as possible automated, everything should be in one place: in a git repository. We want us, teachers, to spend more time on mentoring students, rather than doing repetitive tasks that can be easily automated. Spending time on repetitive tasks makes us lose time, focus and will to educate students.
+RD: Usually content is difficult to update, it is non-digitalized, there is no such thing as versioning, it's difficult to distribute. [Github](https://github.com/) makes it easy to contribute, easy to review, easy to report problems, easy to reuse.
+RD: We use a lot of technologies such as Markdown, reveal.md, draw.io, git, ffmpeg.
+RD: We want the evaluations process to be fully automated. We want to provide an infrastructure where students can practice.
+AO: Where is the Linux Kernel Labs repository?
+RD: Here it is, on [GitHub](https://github.com/linux-kernel-labs).
+Somebody (unknown voice): How much does your work so far depends on the fact that the context is in text format?
+RD: A lot. We want the content to be, ideally, full text format.
+DN: So this means you want people to kick their old PowerPoint Slides / Google Slides / PDFs for this.
+RD: Yes. It will take time and effort to migrate things and get used to it.
+GC: How can we integrate ecuations into the new slides format?
+RD: We have a solution for this. Our partners from Iceland are matematicians and they face this situation as well. Markdown allows you to insert [LaTeX](https://www.latex-project.org/) sections.
+GC: So if I already have then in LaTeX format, it's going to be easy for me?
+RD: Yes, this is what the did.
+DN: You mentioned some tools. Are we limited to those? Can other tools be used?
+RD: Anything is fine. Ideally they should be open source (or free).
+DN: Markdown is used to create other materials as well (e.g. Jupiter Notebook). I have a lot of such notebooks with code and explanations. Is there a way to publice them?
+RD: There are 2 repositories: one with your content and one with the infrastructure that contains a publishing framework of your choice. We currently have Docusaurus for the Operating Systems class. For the Linux Kernel Labs we use Sphinx. Our partners from Iceland use TutorWeb. The storage format is the same. We chose Markdown because it's easy to use. Other formats can be used as well.
```

Comment:
```suggestion
RD: There are 2 repositories: one with your content and one with the infrastructure that contains a publishing framework of your choice. We currently use Docusaurus for the Operating Systems class. For the Linux Kernel Labs, we use Sphinx. Our partners from Iceland use TutorWeb. The storage format is the same. We chose Markdown because it's easy to use. Other formats can be used as well.
```